OpenMEEG/libs/OpenMEEG/src/assembleSensors.cpp

```diff
@@ -117,6 +117,7 @@ namespace OpenMEEG {
         unsigned p0_p1_size = (geo.size() - geo.nb_current_barrier_triangles());
 
         Matrix FergusonMat(3*nbIntegrationPoints, geo.nb_vertices());
+        FergusonMat.set(0.0);
```

Comment:
yes it is ! at least on win10... but I don't know why it was never spotted before.... 
nevertheless, usually, at other places when we declare a matrix, we set it to zero.